php/oop/examples/user.php

```diff
@@ -0,0 +1,28 @@
+<?php
+class User
+{
+
+    public function __construct(public string $email)
```

Comment:
There is no protection for `$email` propery, It's possible to set an invalid email address:
```php
$user = new User('foo@bar.com');
$user->email = 'invalid email';
```